proof/refine/RISCV64/SchedContextInv_R.thy

```diff
@@ -850,7 +850,7 @@ lemma refillNew_corres:
   apply (rule corres_cross_add_guard[where
       Q = "sc_at' sc_ptr and (\<lambda>s'. ((\<lambda>sc. objBits sc = minSchedContextBits + n) |< scs_of' s') sc_ptr)"])
    apply (fastforce dest!: sc_obj_at_cross[OF state_relation_pspace_relation]
-                     simp: obj_at'_def opt_map_red opt_pred_def)
+                     simp: obj_at'_def projectKOs opt_map_red opt_pred_def)
```

Comment:
This points to a slight divergence between `master` and `rt`. `projectKOs` should be in the simp set for more recent architectures, in particular for RISCV. Not relevant to this PR, but worth making an issue for, so we don't forget to address it.

Reply:
It's strange that the ARM version doesn't need it. It looks like we've added the `projectKO` simps to the set for ARM on `rt` and but not on RISCV64

Reply:
`projectKOs` is definitely in the simp set for RISCV64, but not ARM. In this case, it's probably a copying error. I'll remove it.

Reply:
https://github.com/seL4/l4v/issues/687 for the `projectKOs` issue